### ISSUE TITLE
Ignore mailto links

### ISSUE THREAD

**Issue by hugovk** [tcort/markdown-link-check#7]
Would it be possible to have it ignore email addresses?

Right now:

```
[✓] https://twitter.com/
[✓] https://slack.com/
[✖] mailto:test@example.com
```

**Comment by tcort** [comment]
I hadn't considered `mailto` links. Thanks for pointing that out!

I think instead of ignoring `mailto` links, I'll make `markdown-link-check` validate the email address with [isemail](https://www.npmjs.com/package/isemail).

**Comment by hugovk** [comment]
Validation with isemail sounds good!

**Comment by tcort** [comment]
Implemented in version `2.0.0` which has been published to NPM.

**Comment by hugovk** [comment]
Great, thanks!
